### DISCUSSION TITLE
css imports for SSR

### DISCUSSION THREAD

**Discussion by karthage-spartacus** [meteor/meteor#11840] [Feature Requests]
This feature has been asked for a while now (several years), but meteor still lacks css support on the server.
I think it's the missing piece for a full ssr support.

I think we should divide the work into 2 parts : 
- enable css compilation on the server.
- figure out a clever stylesheet collector to serve the stylesheets as raw string in the head.

I've been able to "compile css on the server" on my own using a **hacky** custom package to transform (s)css to a javascript string that I include directly in a style tag server side. It works relatively well, but because it's compiled as javascript it can't be passed down to minifiers / postcss processes, and css modules won't work with this solution (I think).

Maybe a similar process could be used ?
Does anyone used a custom solution to tackle this issue ?

**Comment by Saeeed-B** [comment]
No follow-up for this issue?
